packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -26,6 +27,13 @@ import { feature } from '../../lib/featureManagement';
 import { CustomFeedsExperiment } from '../../lib/featureValues';
 import NotificationsBell from '../notifications/NotificationsBell';
 import classed from '../../lib/classed';
+import { useAuthContext } from '../../contexts/AuthContext';
+
+const OnboardingChecklistBar = dynamic(() =>
+  import(
+    /* webpackChunkName: "onboardingChecklistBar" */ '../checklist/OnboardingChecklistBar'
+  ).then((mod) => mod.OnboardingChecklistBar),
```

Comment:
Why do we use the then mod flow? (just curious)

Reply:
It is used when you don't have default export.